### PR TITLE
Add redux-logger in a development environment

### PR DESCRIPTION
**Pull request by karevn**

### CODE REVIEW

File: app/store/configureStore.js [hunk 1]
```diff
@@ -9,11 +10,17 @@ export default function configureStore(initialState) {
     ? compose
     : window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__; 
 
+  const logger = createLogger({});
```
Comment (nukeop) [inline app/store/configureStore.js]: We already have redux dev tools, which imo provide superior functionality to this logger.

This, however, could be useful if it logged certain important actions to stdout using `electron-timber`.

Comment (karevn) [reply app/store/configureStore.js]: It seems that I've overlooked this miraculous tool. Let me give it a shot.

Comment (karevn) [reply app/store/configureStore.js]: Oh. My. Closing this PR.

Comment (nukeop) [reply app/store/configureStore.js]: Heheh, glad you found out about something useful!

**Pull request closed without merging**